Sources/API/Debug.swift

```diff
@@ -19,119 +19,147 @@ public class Debug {
-        _ target: Any,
+        _ targets: Any...,
+        separator: String = "\n",
         option: Option = Debug.sharedOption,
         to output: inout Target
     ) {
-        Swift.print(_prettyPrint(target, option: option), to: &output)
+        Swift.print(_prettyPrint(targets, separator: separator, option: option), to: &output)
     }
 
-    /// Output debuggable `target` to console.
+    /// Output debuggable `targets` to console.
     /// - Parameters:
-    ///   - target: target
+    ///   - targets: targets
+    ///   - separator: A string to print between each item.
     public static func debugPrint(
-        _ target: Any
+        _ targets: Any...,
+        separator: String = " "
     ) {
-        Swift.print(_debugPrint(target))
+        Swift.print(_debugPrint(targets, separator: separator))
     }
 
-    /// Output debuggable `target` to console.
+    /// Output debuggable `targets` to console.
     /// - Parameters:
-    ///   - target: target
+    ///   - targets: targets
+    ///   - separator: A string to print between each item.
     ///   - output: output
     public static func debugPrint<Target: TextOutputStream>(
-        _ target: Any,
+        _ targets: Any...,
+        separator: String = " ",
         to output: inout Target
     ) {
-        Swift.print(_debugPrint(target), to: &output)
+        Swift.print(_debugPrint(targets, separator: separator), to: &output)
     }
 
-    /// Output debuggable and pretty-formatted `target` to console.
+    /// Output debuggable and pretty-formatted `targets` to console.
     /// - Parameters:
-    ///   - target: target
+    ///   - targets: targets
+    ///   - separator: A string to print between each item.
     ///   - option: option (default: `Debug.sharedOption`)
     public static func debugPrettyPrint(
-        _ target: Any,
+        _ targets: Any...,
+        separator: String = "\n",
         option: Option = Debug.sharedOption
     ) {
-        Swift.print(_debugPrettyPrint(target, option: option))
+        Swift.print(_debugPrettyPrint(targets, separator: separator, option: option))
     }
 
     /// Output debuggable and pretty-formatted `target` to console.
     /// - Parameters:
-    ///   - target: target
+    ///   - targets: targets
+    ///   - separator: A string to print between each item.
     ///   - option: option (default: `Debug.sharedOption`)
     ///   - output: output
     public static func debugPrettyPrint<Target: TextOutputStream>(
-        _ target: Any,
+        _ targets: Any...,
+        separator: String = "\n",
         option: Option = Debug.sharedOption,
         to output: inout Target
     ) {
-        Swift.print(_debugPrettyPrint(target, option: option), to: &output)
+        Swift.print(_debugPrettyPrint(targets, separator: separator, option: option), to: &output)
     }
 
     // MARK: - private
 
     private static func _print(
-        _ target: Any
+        _ targets: [Any],
```

Comment:
Can't use `targets: Any...` because `targets` are passed to **one array of** argument instead of multiple values called from [print() in this source](https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/75/files#diff-14ff193a5989c9da8c6ffb3023d4bfd5R30).

Reply:
That is due to Swift langage design😎